recipes/trimal/1.4/meta.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: trimal
+  version: "1.4.rev15"
+
+source:
+  git_url: https://github.com/scapella/trimal.git
```

Comment:
@gregvonkuster if possible please use the tarballs and a checksum. This makes it easier to backup the sources on depot.